### PR TITLE
Google reCaptcha feature

### PR DESCRIPTION
**Pull request by ShreyasSinha**
Signed-off-by: Shreyas Sinha <shreyas.sinha14@gmail.com>

Solves issue #12983 

Before submitting pull request, please check that every commit:

- [ ] Has proper Signed-Off-By
- [ ] Has commit message which describes it
- [ ] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [ ] Any new functionality is covered by tests

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Yeah, This looks good too.

File: js/functions.js [hunk 1]
```diff
@@ -5022,6 +5022,28 @@ function toggleDatepickerIfInvalid($td, $input_field) {
     }
 }
 
+/*
+* Function to enable the 'Go' button on login.
+*/
+function captcha_enable() {
```
Comment (ibennetch) [inline js/functions.js]: I think this implies enabling the captcha itself, not the "Go" button.

Perhaps a better function name would be `login_go_button_enable()` or something else more descriptive.

File: js/functions.js [hunk 2]
```diff
@@ -5022,6 +5022,28 @@ function toggleDatepickerIfInvalid($td, $input_field) {
...
+    $('#input_go').prop('disabled', false);
+}
+
+/*
+* Function to disable the 'Go' button on login.
+*/
+function captcha_disable() {
```
Comment (ibennetch) [inline js/functions.js]: Likewise, this function name is also a bit misleading.

File: js/functions.js [hunk 3]
```diff
@@ -5022,6 +5022,28 @@ function toggleDatepickerIfInvalid($td, $input_field) {
     }
 }
 
+/*
+* Function to enable the 'Go' button on login.
```
Comment (ibennetch) [inline js/functions.js]: Please align the asterisks by indenting by one space, like
```
/*
 * Function...
 */
```
That way it matches the style of the rest of the file.

**Review by ibennetch — CHANGES_REQUESTED** [review]
When I tried this without reCAPTCHA enabled or configured, the Go button is disabled and I am unable to log in.

It does seem to work as expected when the Captcha is enabled, so nice work on that.

**Comment by edwarddekker** [comment]
Sadly, I am not a programmer. But do have still plenty ideas about new functions on phpMyAdmin.

**Comment by ibennetch** [comment]
@edwarddekker Your input is valuable here. Thanks for taking an interest in improving the project.

To be clear in case I caused confusion, my feedback was specifically meant about the code that ShreyasSinha has proposed. You can feel free to keep making suggestions :)

File: js/functions.js (hunk 3 above)
Comment (ShreyasSinha) [inline js/functions.js]: ok

File: js/functions.js (hunk 1 above)
Comment (ShreyasSinha) [inline js/functions.js]: Yes, that will be better.

**Comment by ShreyasSinha** [comment]
@ibennetch I think this is ready for review.

**Comment by nijel** [comment]
Merged with coding style cleanup in 1f819d8d505aae709c7a9397a82d1cbfbed52c23.

**Comment by ibennetch** [comment]
Thanks @ShreyasSinha

**Pull request merged**